documentation/get-started.adoc

```diff
@@ -136,6 +136,8 @@ You can also run the _Show me a graph_ example Search Phrase yourself by typing
 === Searching for data in Explore
 
 You can use simple search phrases based on your node labels and relationship types to visualize your graph.
+If you ran the example Search Phrase, you should clear the scene before you continue.
+Right-click anywhere on the canvas and select _Clear Scene_ or `Ctrl+Bksp`/`Cmd+Bksp`.
```

Comment:
I'm not sure if we want to overwhelm people in the get-started guide with keyboard shortcuts, while I love them it might be a job of the tool (Explore) to expose them. In the other place, I added it because only with shift you get multi-select.

Reply:
Btw. we should probably use the kbd: macros for keyboard shortcuts (sorry I didn't do it either)
https://docs.asciidoctor.org/asciidoc/latest/macros/keyboard-macro/